wright-filter-wheels/.gitlab-ci.yml

```diff
@@ -43,3 +45,5 @@ twine:
       - dist/*
   only:
     - tags
+
+
```

Comment:
Again, this _whole_ file is not actually used and may be omitted in its entirety

Reply:
deleted as per above